roles/ceph-osd/tasks/scenarios/lvm.yml

```diff
@@ -1,164 +1,17 @@
-  command: "ceph-volume --cluster {{ cluster }} lvm create --filestore --data {{ item.data }} --journal {{ item.journal }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - item.journal_vg is not defined
-    - item.data_vg is not defined
-    - osd_objectstore == 'filestore'
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create filestore osds with dedicated lv journals and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --filestore --data {{ item.data }} --journal {{item.journal_vg }}/{{ item.journal }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - item.journal_vg is defined
-    - item.data_vg is not defined
-    - osd_objectstore == 'filestore'
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create bluestore osds with db and wal and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --bluestore --data {{ item.data }} --block.wal {{ item.wal_vg }}/{{ item.wal }} --block.db {{ item.db_vg }}/{{ item.db }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - osd_objectstore == 'bluestore'
-    - item.data_vg is not defined
-    - item.wal is defined
-    - item.wal_vg is defined
-    - item.db is defined
-    - item.db_vg is defined
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create bluestore osds with db only and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --bluestore --data {{ item.data }} --block.db {{ item.db_vg }}/{{ item.db }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - osd_objectstore == 'bluestore'
-    - item.data_vg is not defined
-    - item.wal is not defined
-    - item.wal_vg is not defined
-    - item.db is defined
-    - item.db_vg is defined
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create bluestore osds with wal only and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --bluestore --data {{ item.data }} --block.wal {{ item.wal_vg }}/{{ item.wal }}"
-  environment:
-    CEPH_VOLUME_DEBUG: 1
-  with_items: "{{ lvm_volumes }}"
-  when:
-    - osd_objectstore == 'bluestore'
-    - item.wal is defined
-    - item.data_vg is not defined
-    - item.wal_vg is defined
-    - item.db is not defined
-    - item.db_vg is not defined
-    - "'{{ item.data }}' not in ceph_volume_lvm_list.stdout"
-
-- name: use ceph-volume to create bluestore osds with just a data device and a raw device or partition for data
-  command: "ceph-volume --cluster {{ cluster }} lvm create --bluestore --data {{ item.data }}"
+- name: "use ceph-volume to create osd: {{ item | to_nice_yaml }}"
```

Comment:
This doesn't seem to expand anything, see: https://2.jenkins.ceph.com/job/ceph-ansible-prs-dev-ansible2.4-bluestore_lvm_osds/4/consoleFull
We can at least reflect  `osd_objectstore`.

Reply:
Yup, I couldn't figure out how to get it to expand anything in ``item`` in ``name``. I should be able to add ``osd_objectstore`` though, I'll do that.